src/main/java/net/comments/sample/TrainingRoom.java

```diff
@@ -1,22 +1,46 @@
 package net.comments.sample;
 
 public class TrainingRoom implements Room {
+    private final String number;
+    private final int size;
```

Comment:
Please remove this variable as it is never used.

Reply:
@extsoft  Are you about size? I used it when create object "small/big room".
For the small room: 
![2018-02-19_2059](https://user-images.githubusercontent.com/36173878/36393337-c9c9f812-15b7-11e8-8ab2-382fbfe48ee9.png)
For the big room: as it should be unlimited, but arrays cannot be unlimited, I've declared the possible size inside of constructor.

Please explain me where I'm wrong?

Reply:
I'm talking about class attribute `size`, not about constructor argument. You declared `private final int size` it, but it's never used in the class implementations.

Reply:
New commit is added. @extsoft  Is it what you mean?